### ISSUE TITLE
Adding zoom to photo's?

### ISSUE THREAD

**Issue by keesvandieren** [kirualex/KASlideShow#53]
We need to be able to add zoom on the photo's in a slideshow.

It should work similar to the Zoom function in the camera app (using double-tab or multi-touch gestures). Any ideas how this can be achieved?

This probably needs changes in the KASlideShow library?

**Comment by kirualex** [comment]
Yes, though it might be a little complex as it would override the current gestures which are used to interact with the slideshow sequencing
